types/time.go

```diff
@@ -576,8 +576,15 @@ func GetFsp(s string) int {
 // GetFracIndex finds the last '.' for get fracStr, index = -1 means fracStr not found.
 // but for format like '2019.01.01 00:00:00', the index should be -1.
 func GetFracIndex(s string) (index int) {
+	tzIndex, _, _, _, _ := GetTimezone(s)
+	var end int
+	if tzIndex != -1 {
+		end = tzIndex
```

Comment:
is char `s[tzIndex]` a part of datetime? since the for following for-loop begin at `tzIndex` not `tzIndex-1`

Reply:
have you test it locally？

Reply:
yes